### PR TITLE
Various map fixes

### PR DESCRIPTION
**Pull request by Iglpigl**
## About The Pull Request

Changes all roundstart monkeys to monkey cubes.
Punpun is in his own special cube which specifically spawns him.
Also puts one more artist landmark on map so that instead of when having two artists ready roundstart one spawns in office and one in in cryo now both spawn in the office.
Also deletes a floating camera in medbay isolation.

## Why It's Good For The Game

Dev del approval.

## Changelog
:cl:
tweak: roundstart monkeys changed to monkey cubes, artists now spawn in office always instead of only when there is one artist and floating camera in medbay isolation deleted
/:cl:

### CODE REVIEW

**Comment by Holvr0** [comment]
I am Holvr and I approve of this change.

**Comment by Humonitarian** [comment]
I am Humon and I approve of this change.

**Comment by MLGTASTICa** [comment]
I am SPCR and this does barely anything to fix lag.

**Comment by Alliostra** [comment]
I'm Commander Shepard, and This Is My Favorite change on the github

**Comment by ErisMeleeEnjoyer** [comment]
I'm ErisMeleeEnjoyer, and this is the best borer nerf on github.

**Comment by SirRichardFrancis** [comment]
I am KIROV and I am expecting Pun-Pun.

**Review by Firefox13 — APPROVED** [review]
that's so funny actually when in monkey cell there's a bunch of cubes instead

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -1810,6 +1810,17 @@
 	preloaded_reagents = list("protein" = 10)
 	taste_tag = list(MEAT_FOOD,SALTY_FOOD)
 
+/obj/item/reagent_containers/food/snacks/monkeycube/punpun
+    name = "emergency companion cube"
+
+/obj/item/reagent_containers/food/snacks/monkeycube/punpun/Expand()
+    src.visible_message(SPAN_NOTICE("\The [src] expands!"))
```
Comment (SirRichardFrancis) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1817]:
```suggestion
    visible_message(SPAN_NOTICE("\The [src] expands!"))
```

**Comment by SirRichardFrancis** [comment]
That's one hell of title you've got here. Consider moving all that additional info in the description.

**Pull request merged**